src/App.jsx

```diff
@@ -15,14 +27,24 @@ function App() {
   };
 
   return (
-    <ChakraProvider>
-      <DropZone setFiles={setFiles} />
-      <Button onClick={onSubmit}>Upload File</Button>
-      {images.map(e => (
-        <Image key={e} src={e} />
-      ))}
-      <DonationForm />
-    </ChakraProvider>
+    <Router>
+      <ChakraProvider>
+        <DropZone setFiles={setFiles} />
+        <Button onClick={onSubmit}>Upload File</Button>
+        {images.map(e => (
+          <Image key={e} src={e} />
+        ))}
+        <DonationForm />
+      </ChakraProvider>
+      <Routes>
+        <Route path="/" element={<Dashboard />} />
+        <Route path="/donate/edit" element={<DonateEdit />} />
+        <Route path="/drivers" driver={driverOne} element={<Drivers />} />
+        <Route path="/routes" route={routeOne} element={<RoutesPage />} />
+        <Route path="/donate" element={<Donate />} />
```

Comment:
for the `/drivers/:driverId` and `/routes/:routeId` routes u should use url params instead ([example here](https://v5.reactrouter.com/web/example/url-params))